pkg/persistence/config/writer/config_writer.go

```diff
@@ -769,11 +772,12 @@ func newDetailedConfigWriterError(context *serializerContext, err error) configE
 	}
 }
 
-func fmtDetailedConfigWriterError(context *serializerContext, format string, args ...interface{}) configError.DetailedConfigWriterError {
+func fmtDetailedConfigWriterError(context *serializerContext, err error) configError.DetailedConfigWriterError {
```

Comment:
Can we remove this and just use the function above, `newDetailedConfigWriterError(...)`? It now looks just the same I think.

Reply:
Awesome catch, did not see that one